recipes/exprtk/all/conanfile.py

```diff
@@ -1,42 +1,51 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.layout import basic_layout
+from conan.tools.files import get, copy, load, save
 import os
-from conans import ConanFile, tools
 
-required_conan_version = ">=1.33.0"
+required_conan_version = ">=1.53.0"
 
 
 class ExprTkConan(ConanFile):
     name = "exprtk"
     description = "C++ Mathematical Expression Parsing And Evaluation Library ExprTk"
-    license = ("MIT")
-    topics = ("exprtk", "cpp", "math", "mathematics", "parser", "lexer", "numerical")
+    license = "MIT"
+    topics = ("math", "mathematics", "parser", "lexer", "numerical")
     homepage = "https://www.partow.net/programming/exprtk/index.html"
     url = "https://github.com/conan-io/conan-center-index"
     settings = "compiler"
     no_copy_source = True
 
     @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    def _min_cppstd(self):
+        return 17
```

Comment:
Is it required?
I find following information from github readme.

```
ExprTk has been built error and warning free using the following set
of C++ compilers:

  (*) GNU Compiler Collection (3.5+)
  (*) Intel C++ Compiler (8.x+)
  (*) Clang/LLVM (1.1+)
  (*) PGI C++ (10.x+)
  (*) Microsoft Visual Studio C++ Compiler (8.1+)
  (*) IBM XL C/C++ (9.x+)
  (*) C++ Builder (XE4+)
```

Reply:
hmm it was originally C++11 as a minimum, I'll change it back.
Not sure why I changed it to 17 ...

Reply:
Note that original recipe author is also library author, so I am assuming C++11 is the correct minimum!